Formula/g2s.rb

```diff
@@ -0,0 +1,74 @@
+class G2s < Formula
+  desc "Toolbox for geostatistical simulations."
+  homepage "https://gaia-unil.github.io/G2S/"
+  url "https://github.com/GAIA-UNIL/g2s/archive/0acc033d06e7a3ac5ae4f13def02a43cec0adefe.tar.gz"
+  sha256 "7a95b04057bbdbbb11d37d4a6131d0b60981822c3e60e7986ea287b926b734ba"
+  license "GPL-3.0-only"
+  version "0.98.015"
+
+  # Add dependencies
+  depends_on "zlib"
+  depends_on "fftw"
+  depends_on "zeromq"
+  depends_on "libomp"
+  depends_on "jsoncpp"
+  depends_on "cppzmq"
+  depends_on "curl"
+
+   def install
+
+    cd "build" do
+      # Run "make c++ -j"
+      system "make", "c++", "-j", "CXXFLAGS=-Xclang -fopenmp -I#{Formula["fftw"].opt_include}", "LIB_PATH=-L#{Formula["fftw"].opt_lib} -L#{Formula["libomp"].opt_lib} -lomp"
+    end
+
+    # Copy g2s_server and other from the c++-build folder to the brew bin folder
+    (bin).install "build/g2s-package/g2s-brew/g2s" 
+    (prefix/"g2s_bin").install "build/c++-build/g2s_server" 
+    (prefix/"g2s_bin").install "build/c++-build/echo" 
+    (prefix/"g2s_bin").install "build/c++-build/qs" 
+    (prefix/"g2s_bin").install "build/c++-build/nds" 
+    (prefix/"g2s_bin").install "build/c++-build/ds-l" 
+    (prefix/"g2s_bin").install "build/c++-build/errorTest" 
+    (prefix/"g2s_bin").install "build/c++-build/auto_qs"
+    (prefix/"g2s_bin").install "build/algosName.config"
```

Comment:
is there no `make install`?

Reply:
A folder in the `bin` directory means that anything in the folder won't be picked up in the users PATH, so that doesn't seem like a good idea either way.

Reply:
I suppose we can debate about this it is some executable that are run by the server, and therefore do not need to be in the main path. Even the server itself is not expected to be called by the user. That the reason why have a script to handle all kind of request

Reply:
Can we install these into `libexec` instead of a custom directory?

Reply:
this is a very good idea, let me do that